benchmark/ui.js

```diff
@@ -35,13 +36,44 @@ function updateOpsSelect() {
   supportedOps = getSelectedOps();
 }
 
+async function setSuperImageUI(modelClass, modelName) {
+  if (modelClass === 'super_resolution') {
+    imageElement.parentNode.setAttribute('align', '');
+    imageElement.style.width = '300px';
+    imageElement.style.height = '300px';
+    let srCanvas = document.createElement('canvas');
+    let srCtx = srCanvas.getContext('2d');
+    switch (modelName) {
+      case 'SRGAN 96x4 (TFLite)':
+        srCanvas.width = 96;
+        srCanvas.height = 96;
+        break;
+      case 'SRGAN 128x4 (TFLite)':
+        srCanvas.width = 128;
+        srCanvas.height = 128;
+        break;
+      default:
+        srCanvas.width = 96;
+        srCanvas.height = 96;
+    }
```

Comment:
Please, i prefer to use CSS rather than javascript for this kind of work, thoughts?